docs/website/docs/dlt-ecosystem/verified-sources/sql_database.md

```diff
@@ -9,9 +9,10 @@ or [book a call](https://calendar.app.google/kiLhuMsWKpZUpfho6) with our support
 SQL databases are management systems (DBMS) that store data in a structured format, commonly used
 for efficient and reliable data retrieval.
 
-This SQL database `dlt` verified source and
-[pipeline example](https://github.com/dlt-hub/verified-sources/blob/master/sources/sql_database_pipeline.py)
-loads data using SqlAlchemy to the destination of your choice.
+Our SQL Database verified source loads data to your specified destination using SQLAlchemy.
+
+> View the pipeline example
```

Comment:
I suggest using a tip [admonition](https://docusaurus.io/docs/markdown-features/admonitions) instead of the markdown blockquote.